### DISCUSSION TITLE
GitHub Universe is ready to launch 🚀 🪐 Tune in Now!

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#74675] [Announcements]
![LinkedIn banner_1584x396](https://github.com/community/community/assets/4041275/6f278374-0147-464b-9db8-6bef6cb23a7b)

## [Register Now!](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=launch) for free to attend virtually today

We have an exciting, jam packed day for you here at GitHub Universe and **you’re just in time! 🕧**

## Where can we watch?

All [livestreaming sessions](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=Keynote&search.deliveryformat=1692799009854003CF33) will become available on our[ GitHub Universe website](https://githubuniverse.com/) soon.

[Watch it Live Here](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/watch)

Selected livestream sessions will be available on other streaming platforms ([YouTube](https://www.youtube.com/watch?v=NrQkdDVupQE) & [Linkedin](https://www.linkedin.com/events/githubuniverse2023-day17124783508178313216/comments/) ) 📺

So grab a beverage and some nourishment ☕🥝 and get ready for some exciting GitHub updates all from the comfort of your seat! **But watch out**, you may just fall off it with all the incredible news. 🤯

All sessions that are not live streamed will be uploaded approximately 2 hours after they finish. 

## Where can we discuss?

[Right here!](https://github.com/orgs/community/discussions/) 👋 The [GitHub Community](https://github.com/orgs/community/discussions/) has dedicated categories for each of the GitHub Universe tracks. Some of our **speakers** have already created discussions for you to ask them directly about their session’s topic. Here’s your chance to connect and discover more.

🤖 [AI - Let’s discuss a new frontier in developer creativity](https://github.com/orgs/community/discussions/categories/ai)

🔒 [Security - Let’s discuss proactive security, powered by AI](https://github.com/orgs/community/discussions/categories/security)

🚀 [DevEx (Developer Experience) - Let’s discuss your flow state.](https://github.com/orgs/community/discussions/categories/devex)

## Keynote 🔑

It’s a new age for GitHub. Join our CEO Thomas Dohmke and special guests for a stirring, high velocity keynote as we fundamentally redefine what it means to be a software development platform in the age of AI.

[Watch the Keynote for Day 1 here](https://www.youtube.com/watch?v=NrQkdDVupQE)

We'll also explore how the best companies, developers, and organizations are using GitHub to transform industries and are setting the stage for a more optimistic future. 

[Day 1 Keynotes on YouTube](https://www.youtube.com/watch?v=NrQkdDVupQE) 📺

Let us know what sessions you're attending or excited for 🎤 !

**Comment by Sokhanetaze80** [comment]
12ib7dApVFvg82TXKycWBNpN8kFyiAN1dr

**Comment by ghostinhershell** [comment]
Really excited for the [The future state of APIs [DEV2784S]](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1694819271085001rTHZ) talk happening later today! 🚀

**Reply by nicknijenhuis** [reply]
Is there any way to watch this on YouTube so that I can cast it?

**Reply by mishmanners** [reply]
Yes, you can watch the keynote here: https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1695657268465001jL1D and live streams are under the "Live" tab on YouTube. As sessions are completed, they'll be uploaded onto the githubuniverse.com website approx. 2 hours after the conclusion of the session. These are YouTube embedded videos so you should be able to cast it.

**Comment by emil-nasso** [comment]
Is there no way to cast this to a tv? Player doesnt seem to have support for chromecast or airplay.

**Reply by samus-aran** [reply]
We are also[ livestreaming on YouTube](https://www.youtube.com/@GitHub/streams) that you could cast ❤️

**Reply by emil-nasso** [reply]
Nice! That is so much better! Thanks.

**Reply by bajoski34** [reply]
also [streaming on linkedin](https://www.linkedin.com/events/githubuniverse2023-day17124783508178313216/theater/)

**Reply by nicknijenhuis** [reply]
I cannot find it on youtube, could you share the link to the actual livestream?

**Reply by mishmanners** [reply]
Yes, you can watch the keynote here: https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1695657268465001jL1D and live streams are under the "Live" tab on YouTube. As sessions are completed, they'll be uploaded onto the githubuniverse.com website approx. 2 hours after the conclusion of the session. These are YouTube embedded videos so you should be able to cast it.

**Comment by DarkteK** [comment]
Is there any way to obtain a Certification test voucher at this point?

**Reply by ettaboyle** [reply]
Great question! I don't believe so. I'm digging around for more info for and will update here if I find anything.

**Reply by mishmanners** [reply]
GitHub Certifications will be coming soon. Certifications are pre-available to all those who are attending GitHub Universe in person. You can find all the information here: https://resources.github.com/learn/certifications/. Certifications will be available to everyone soon. Keep your eyes posted.

**Comment by smach** [comment]
Is there some way to turn off the close captions? They are interfering with trying to read the code on the screen

**Reply by ettaboyle** [reply]
You can toggle them off if you're watching the livestream on YouTube

**Reply by smach** [reply]
Will all the sessions be on YouTube too or just the keynote?

**Reply by ettaboyle** [reply]
All sessions that are not live streamed will be uploaded to YouTube approximately 2 hours after they finish. We're also asking speakers to post discussions and those will link to session recordings. You can check out the info in the top of this discussion for info on all the tracks.

> Where can we discuss?
> [Right here!](https://github.com/orgs/community/discussions/) 👋 The [GitHub Community](https://github.com/orgs/community/discussions/) has dedicated categories for each of the GitHub Universe tracks. Some of our speakers have already created discussions for you to ask them directly about their session’s topic. Here’s your chance to connect and discover more.
> 
> 🤖 [AI - Let’s discuss a new frontier in developer creativity](https://github.com/orgs/community/discussions/categories/ai)
> 
> 🔒 [Security - Let’s discuss proactive security, powered by AI](https://github.com/orgs/community/discussions/categories/security)
> 
> 🚀 [DevEx (Developer Experience) - Let’s discuss your flow state.](https://github.com/orgs/community/discussions/categories/devex)

**Reply by Nekall** [reply]
I came here to ask the same question, thanks for the answer ! 🙏

**Reply by mishmanners** [reply]
The  keynote is now available here: https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1695657268465001jL1D and live streams are under the "Live" tab on YouTube. As sessions are completed, they'll be uploaded onto the githubuniverse.com website approx. 2 hours after the conclusion of the session. These are YouTube embedded videos so you should be able to cast it.

**Comment by nicknijenhuis** [comment]
> We are also[ livestreaming on YouTube](https://www.youtube.com/@GitHub/streams) that you could cast ❤️

Are you sure? I don't see anything on YouTube. Some stream with ChromeCast support would be really nice

**Reply by samus-aran** [reply]
We've updated the post to provide clarification on where the livestream is happening and the uploaded videos. ✨

**Comment by ChrisArchitect** [comment]
Not so helpful answering all the requests for a stream here with "we're on YouTube" but there's no livestream available on there... only the Day 2 scheduled stream.

**Reply by samus-aran** [reply]
We've updated the post to provide clarification on where the livestream is happening and the uploaded videos. ✨

**Comment by nicomulderlawvu1** [comment]
Where can one re-watch the opening? it doesn't seem possible on the attendee portal... or re-watch a scheduled session?

**Reply by nicomulderlawvu1** [reply]
found it, thanks

**Reply by gitstua** [reply]
Hi @nicomulderlawvu1 - You can watch the keynote in this YouTube link - https://www.youtube.com/live/h3Bwuzz0TNA?feature=shared&t=698

Just scroll the timeline to any section you are looking for 🎉

**Comment by mishmanners** [comment]
Yassss! Where are you all tuning in from?

**Reply by jricciardi** [reply]
👋 In from Seattle over here!

**Reply by mishmanners** [reply]
👋 noice. I'm tuning in from Australia!

**Reply by gitstua** [reply]
👋🏽  Hi from Australia! 🦘

**Reply by Nekall** [reply]
👋🏻 Hi from France 🥐🥖

**Reply by gitstua** [reply]
Bonjour @Nekall! 🇫🇷🇫🇷🇫🇷

**Reply by Nekall** [reply]
Bonjour @gitstua 😋

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
